sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -540,16 +700,205 @@ def from_predictions(
+            parameters are applied to the precision-recall curves of each CV fold
+            sequentially. If a single dictionary is provided, the same
+            parameters are applied to all precision-recall curves.
+
+        plot_chance_level : bool, default=False
+            Whether to plot the chance level lines.
+
+        chance_level_kwargs : dict, default=None
+            Keyword arguments to be passed to matplotlib's `plot` for rendering
+            the chance level lines.
+
+        despine : bool, default=False
+            Whether to remove the top and right spines from the plot.
+
+        Returns
+        -------
+        display : :class:`~sklearn.metrics.PrecisionRecallDisplay`
+
+        See Also
+        --------
+        PrecisionRecallDisplay.from_predictions : Plot precision-recall curve
+            using estimated probabilities or output of decision function.
+        PrecisionRecallDisplay.from_estimator : Plot precision-recall curve
+            using an estimator.
+        precision_recall_curve : Compute precision-recall pairs for different
+            probability thresholds.
+        average_precision_score : Compute average precision (AP) from prediction scores.
+
+        Notes
+        -----
+        The average precision (cf. :func:`~sklearn.metrics.average_precision_score`)
+        in scikit-learn is computed without any interpolation. To be consistent
+        with this metric, the precision-recall curve is plotted without any
+        interpolation as well (step-wise style).
+
+        You can change this style by passing the keyword argument
+        `drawstyle="default"`. However, the curve will not be strictly
+        consistent with the reported average precision.
+
+        Examples
+        --------
+        >>> import matplotlib.pyplot as plt
+        >>> from sklearn.datasets import make_classification
+        >>> from sklearn.metrics import PrecisionRecallDisplay
+        >>> from sklearn.model_selection import cross_validate
+        >>> from sklearn.svm import SVC
+        >>> X, y = make_classification(random_state=0)
+        >>> clf = SVC(random_state=0)
+        >>> cv_results = cross_validate(
+        ...     clf, X, y, cv=3, return_estimator=True, return_indices=True)
+        >>> PrecisionRecallDisplay.from_cv_results(cv_results, X, y)
+        <...>
+        >>> plt.show()
+        """
+        cls._validate_from_cv_results_params(
+            cv_results, X, y, sample_weight=sample_weight
+        )
+
+        precision_folds, recall_folds, ap_folds, prevalence_pos_label_folds = (
+            [],
+            [],
+            [],
+            [],
+        )
+        for estimator, test_indices in zip(
+            cv_results["estimator"], cv_results["indices"]["test"]
+        ):
+            y_true = _safe_indexing(y, test_indices)
+            y_pred, pos_label_ = _get_response_values_binary(
```

Comment:
Not sure if there needs to be any note/comment here.
`_get_response_values_binary` infers `pos_label_` to be `classes[-1]` when `pos_label=None`, which is fine unless:

* data is binary but one class is missing in a cv fold - in this case the estimator will raise an error that it was fit on data that only contains one class
* data is multi-class - `precision_recall_curve` will complain